build.gradle

```diff
@@ -145,3 +146,22 @@ uploadArchives {
         }
     }
 }
+
+publishing {
+  publications {
+    maven(MavenPublication) {
+      groupId = 'com.gocardless'
+      artifactId = 'gocardless-pro'
+    }
+  }
+  repositories {
+    maven {
+      name "OSSRH"
+      url "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
+      credentials {
+        username System.getenv("MAVEN_USERNAME")
```

Comment:
Need to add this secret in Github

Reply:
@ankithads do we know how to do this?
Are there any other steps required to make this work? (other than actually testing the thing)

Reply:
@paprikati Sorry for the late reply.
Some one with admin access to git hub can add it.
Yes, after testing we will be able to figure out if anything is required

Reply:
cool. Are you happy to start testing or do you need some support from me?

Reply:
Will be able to start testing may be in 2 weeks. I would like to test it with the minor patch. haven't done any changes to the public API lately. If you are doing any minor changes in 2 weeks we could test with those changes. hoping nothing breaks 😄  🤞